crawl-ref/source/dat/strings/de/annotations.txt

```diff
@@ -151,28 +151,28 @@ MR
 
 %%%%
 resist magic
-Magie resistieren
+Magie widerstehen
 %%%%
 resist cold
-Kälte resistieren
+Kälte widerstehen
 %%%%
 resist corrosion
-Korrosion resistieren
+Korrosion widerstehen
 %%%%
 resist electricity
-Elektrizität resistieren
+Elektrizität widerstehen
 %%%%
 resist fire
-Feuer resistieren
```

Comment:
Can we make these "Feuerresistenz", etc. because then it matches better with the abbreviations rFeuer, etc.?